.travis.yml

```diff
@@ -40,18 +43,18 @@ install:
   - nvm install $TRAVIS_NODE_VERSION;
   - npm install
   - npm run vscode:prepublish
-  - if [[ "$(go version)" =~ "go version go1.8" ]]; then go get -u -v github.com/nsf/gocode; else go get -u -v github.com/mdempsky/gocode; fi
-  - go get -u -v github.com/rogpeppe/godef
-  - if [[ "$(go version)" =~ "go version go1.8" ]]; then echo skipping gogetdoc; else go get -u -v github.com/zmb3/gogetdoc; fi
-  - if [[ "$(go version)" =~ "go version go1.8" ]]; then echo skipping golint; else go get -u -v golang.org/x/lint/golint; fi
-  - go get -u -v github.com/ramya-rao-a/go-outline
-  - go get -u -v github.com/sqs/goreturns
-  - go get -u -v golang.org/x/tools/cmd/gorename
-  - go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs
   - go get -u -v github.com/acroca/go-symbols
   - go get -u -v github.com/cweill/gotests/...
-  - go get -u -v github.com/haya14busa/goplay/cmd/goplay
   - go get -u -v github.com/davidrjenni/reftools/cmd/fillstruct
+  - go get -u -v github.com/haya14busa/goplay/cmd/goplay
+  - go get -u -v github.com/mdempsky/gocode
+  - go get -u -v github.com/ramya-rao-a/go-outline
+  - go get -u -v github.com/rogpeppe/godef
+  - go get -u -v github.com/sqs/goreturns
+  - go get -u -v github.com/uudashr/gopkgs/cmd/gopkgs
+  - go get -u -v github.com/zmb3/gogetdoc
+  - go get -u -v golang.org/x/lint/golint
+  - go get -u -v golang.org/x/tools/cmd/gorename
```

Comment:
I removed the checks for Go 1.8, which we're not testing anymore. I also sorted this list.